app/src/main/java/org/fossasia/openevent/app/data/attendee/AttendeeRepositoryImpl.java

```diff
@@ -43,6 +46,8 @@ public AttendeeRepositoryImpl(Repository repository, AttendeeApi attendeeApi) {
         // There is no use case where we'll need to load single attendee from network
         return repository.observableOf(Attendee.class)
             .reload(reload)
+            .rateLimiter(rateLimiter)
+            .rateLimiterKey("Attendee")
```

Comment:
These both arguments are coupled, take them using one function in builder in order key, rateLimiter

Nice job, LGTM otherwise